<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editor</class>
 <widget class="QFrame" name="editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="NLineEditMouse" name="titleEditor"/>
   </item>
   <item>
    <widget class="NTextEdit" name="textEditor"/>
   </item>
   <item>
    <widget class="QFrame" name="bottomArea">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="NLineEditMouse" name="tagEditor">
          <property name="placeholderText">
           <string>Tags separated by space</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="margin">
         <number>9</number>
        </property>
        <item>
         <widget class="QPushButton" name="dtBtn">
          <property name="toolTip">
           <string>Click to edit</string>
          </property>
          <property name="icon">
           <iconset resource="../../res/res.qrc">
            <normaloff>:/clock.png</normaloff>:/clock.png</iconset>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="lockBtn">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>Click to turn off read-only mode</string>
          </property>
          <property name="icon">
           <iconset resource="../../res/res.qrc">
            <normaloff>:/lock.png</normaloff>:/lock.png</iconset>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QDialogButtonBox" name="box">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="standardButtons">
           <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
          </property>
          <property name="centerButtons">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NTextEdit</class>
   <extends>QTextEdit</extends>
   <header>hazama.ui.customwidgets</header>
  </customwidget>
  <customwidget>
   <class>NLineEditMouse</class>
   <extends>QLineEdit</extends>
   <header>hazama.ui.customwidgets</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../res/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>box</sender>
   <signal>accepted()</signal>
   <receiver>editor</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>box</sender>
   <signal>rejected()</signal>
   <receiver>editor</receiver>
   <slot>closeNoSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>closeNoSave()</slot>
  <slot>updateTagEditorFont()</slot>
 </slots>
</ui>
